src/v2/Apps/_WorksForYou2/WorksForYou2App.tsx

```diff
@@ -1,9 +1,160 @@
-import { Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MetaTags } from "v2/Components/MetaTags"
+import { WorksForYou2App_viewerArtist } from "v2/__generated__/WorksForYou2App_viewerArtist.graphql"
+import { WorksForYou2App_viewerFeed } from "v2/__generated__/WorksForYou2App_viewerFeed.graphql"
+import { WorksForYou2App_viewerMe } from "v2/__generated__/WorksForYou2App_viewerMe.graphql"
+import { WorksForYou2App_viewerSidebarAggregations } from "v2/__generated__/WorksForYou2App_viewerSidebarAggregations.graphql"
+import { WorksForYou2FeedPaginationContainer } from "./Components/WorksForYou2Feed"
+import {
+  Column,
+  FilterSelect,
+  FilterSelectChangeState,
+  FilterSelectItems,
+  GridColumns,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { WorksForYou2ArtistFeedPaginationContainer } from "./Components/WorksForYou2ArtistFeed"
+import { useRouter } from "v2/System/Router/useRouter"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { RouterLink } from "v2/System/Router/RouterLink"
+
+interface WorksForYou2Props {
+  viewerArtist: WorksForYou2App_viewerArtist
+  viewerFeed: WorksForYou2App_viewerFeed
+  viewerMe: WorksForYou2App_viewerMe
+  viewerSidebarAggregations: WorksForYou2App_viewerSidebarAggregations
+}
+
+const WorksForYou2App: React.FC<WorksForYou2Props> = ({
+  viewerArtist,
+  viewerFeed,
+  viewerMe,
+  viewerSidebarAggregations,
+}) => {
+  const { router } = useRouter()
+
+  const artists =
+    (viewerSidebarAggregations.sidebarAggregations?.aggregations?.[0]
+      ?.counts as FilterSelectItems) ?? []
+
+  const savedArtworks = extractNodes(
+    viewerMe.me?.followsAndSaves?.bundledArtworksByArtistConnection
+  )
+
+  const handleResultsFilterChange = (state: FilterSelectChangeState) => {
+    const artistSlug = state.selectedItems[0]?.value
+
+    if (artistSlug) {
+      router.push(`/works-for-you2/${artistSlug}`)
+    } else {
+      router.push("/works-for-you2")
+    }
+  }
 
-export const WorksForYou2App: React.FC = () => {
   return (
     <>
-      <Text variant="xl">Works For You 2 App</Text>
+      <MetaTags title="Works For You | Artsy" description="TODO" />
```

Comment:
TODO

Reply:
The title should be `"Works by Artists You Follow | Artsy"`, and I don't think there is a description.